online-shop/src/index.css

```diff
@@ -0,0 +1,102 @@
+.productDetails {
```

Comment:
Pune clasele de css in fisiere separate pentru fiecare componenta, ca ti se mareste foarte tare fisierul asta si nu o sa mai intelegi ce-i pe aici.